src/query/ast/src/parser/token.rs

```diff
@@ -1094,3 +1096,13 @@ impl TokenKind {
         }
     }
 }
+
+pub fn all_reserved_keywords() -> String {
```

Comment:
`all_reserved_keywords()` should returns `Vec<String>`, and convert to the required format where it is used.

Reply:
Thanks a lot.